packages/compass-aggregations/src/components/pipeline-explain/explain-query-performance.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Body, Subtitle, css, spacing } from '@mongodb-js/compass-components';
+import type { IndexInformation } from '@mongodb-js/explain-plan-helper';
+
+import { ExplainIndexes } from './explain-indexes';
+
+type ExplainQueryPerformanceProps = {
+  executionTimeMillis: number;
+  nReturned: number;
+  usedIndexes: IndexInformation[];
+};
+
+const containerStyles = css({
+  display: 'flex',
+  gap: spacing[3],
+  flexDirection: 'column',
+});
+
+const statsStyles = css({
+  gap: spacing[1],
+  display: 'flex',
+  flexDirection: 'column',
+});
+
+const statItemStyles = css({
+  display: 'flex',
+  gap: spacing[1],
+});
+
+const statTitleStyles = css({
+  whiteSpace: 'nowrap',
+});
+
+export const ExplainQueryPerformance: React.FunctionComponent<ExplainQueryPerformanceProps> =
+  ({ nReturned, executionTimeMillis, usedIndexes }) => {
+    return (
+      <div
+        className={containerStyles}
+        data-testid="pipeline-explain-results-summary"
+      >
+        <Subtitle>Query Performance Summary</Subtitle>
+        <div className={statsStyles}>
+          <div className={statItemStyles}>
+            <Body>Documents returned:</Body>
+            <Body weight="medium">{nReturned}</Body>
+          </div>
+          <div className={statItemStyles}>
+            <Body>Actual query execution time(ms):</Body>
+            <Body weight="medium">{executionTimeMillis}</Body>
```

Comment:
Even on a very long running aggregation I'm getting a zero execution time, not sure if that's expected (also no `nReturned`):

<img width="632" alt="image" src="https://user-images.githubusercontent.com/5036933/169001179-a8b54e78-2ec9-457e-a4af-d941437adf50.png">

This is from running following aggregation on `Corporate-Registrations.corporations`

```json
[
    {
        "$sort": {
            "name": -1
        }
    },
    {
        "$limit": 100
    }
]
```

Reply:
hmm. great catch! i didn't find this scenario when i was working on processing raw server explain in my previous tickets. also since the raw explain is not documented anywhere, its hard to handle all the cases.

in this case, i'll hide the section if we don't have any info, while I work on fixing it in another pr.

Reply:
Added a fix in #3170